test_suite/generic/test_generic.py

```diff
@@ -172,6 +171,18 @@ def test_release_version(self, host, instance_data, rhel_only):
 
         assert product_version == package_release_version, \
             f'product version ({product_version}) does not match package release version'
+
+    @pytest.mark.pub
+    def test_release_version_in_image_name(self, host, instance_data, rhel_only):
```

Comment:
Not needed here now that the original test has been decoupled.

```suggestion
    def test_release_version_in_image_name(self, host, instance_data):
```

Reply:
Great!